### PR TITLE
Update configure-event-hub.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed typo

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8419

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/configure-event-hub.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Learn how to configure your Event Hub so that it can ingest events from Microsof
     :::image type="content" source="../../media/759498162a4e93cbf17c4130d704d164.png" alt-text="The event hubs properties section in the Microsoft Azure portal" lightbox="../../media/759498162a4e93cbf17c4130d704d164.png":::
 
 
-1. Once the Event Hub Namespace is created, you will need to add the App Registration Service Principal as Reader, Azure Event Hub Data Receiver, and the user who will be logging into Microsoft 365 Defender as Contributor (you can also do this at Resource Group or Subscription level).
+1. Once the Event Hub Namespace is created, you will need to add the App Registration Service Principal as Reader, Azure Event Hubs Data Receiver, and the user who will be logging into Microsoft 365 Defender as Contributor (you can also do this at Resource Group or Subscription level).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/configure-event-hub.md:80]:
```suggestion
1. Once the Event Hubs namespace is created, you will need to add the App Registration Service Principal as Reader, Azure Event Hubs Data Receiver, and the user who will be logging into Microsoft 365 Defender as Contributor (you can also do this at Resource Group or Subscription level).
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**